tests/yam/verify_consistency_self_update.pm

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 SUSE LLC
+#
+# Summary: check that self updates can be correctly installed on a
+#          GM installation.
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    select_serial_terminal;
+
+    zypper_call 'ar -f http://dist.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-INSTALLER:/15-SP5:/x86_64/update/ self-update';
+    # enable all repos so that we can install yast later on
+    zypper_call 'mr -e --all';
```

Comment:
I don't get this part, could you please clarify?

Reply:
leftover from previous testing, removed